components/chainhook-cli/src/storage/signers.rs

```diff
@@ -143,7 +143,7 @@ pub fn initialize_signers_db(base_dir: &PathBuf, ctx: &Context) -> Result<Connec
         "CREATE TABLE IF NOT EXISTS mock_signatures (
             id INTEGER PRIMARY KEY AUTOINCREMENT,
             mock_proposal_id INTEGER NOT NULL,
-            message_id INTEGER,
+            message_id INTEGER NOT NULL,
```

Comment:
I think these need to be added using migration / alter-table logic otherwise they won't apply to existing dbs

Reply:
@zone117x Are those migrations on `signers-metrics-api`?

Reply:
No the change in this PR updates this sqlite schema but those updates will only apply if starting with a fresh database. If this PR is ran against the current chainhook instance then the tables already exist and this change won't have any effect.

Reply:
Ok. Resolving this conversation and will change the implementation to a graceful error handling.